packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -647,6 +647,7 @@ export const FLAGS: Record<string, FlagInfo> = {
     compatibilityWithOldBehaviorMd: 'Set installLatestAwsSdk: true on all resources that need it.',
     introducedIn: { v2: '2.60.0' },
     recommendedValue: false,
+    defaults: { v2: true }
```

Comment:
```suggestion
    defaults: { v2: true },
```

To fix the build error